### PR TITLE
Feature/hero-img/Add hero image

### PR DESCRIPTION
**Pull request by kacus**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/8d156b6924.js" crossorigin="anonymous"></script>
 </head>
 
 <body>
-
-<p>CodersCamp - projekt 1. Tutaj możesz zacząć swoją zabawę!</p>
-
+    <div class="hero">
+        <div class="hero__logo">
```
Comment (Skorcese) [inline index.html:14]: The title is not centered on all viewports. That's caused by .hero__logo content overflowing their span container due to **letter-spacing: -1rem;**

You can either set the letter spacing to positive values and adjust the position of characters manually, or move the whole section with transform: translateX(...);

image

image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
+/* FONTS' IMPORTS */
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+
```
Comment (Skorcese) [inline style.css:3]: I can see you are using relative units already, but this trick could let you measure rems in relation to pixels - 1:1

html{
font-size: 62.5%;
}

body{
font-size: 1.6rem;
}

This way you can write 123px as 12.3rem for example
https://developer.mozilla.org/en-US/docs/Web/CSS/font-size

File: style.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+/* HERO - LINK */
+.hero__link {
+    text-align: center;
+    margin-top: auto;
+    padding   : 1em;
+}
+
+/* MOBILE LANDSCAPE */
+@media screen and (min-width: 480px) {
```
Comment (Skorcese) [inline style.css:78]: @media screen and (min-width: 480px) and (orientation: landscape) { ... }

File: style.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    /* HERO - INTRODUCTION */
+    .hero__introduction {
+        width: auto;
+
+        border-radius: 1em;
+    }
+}
+
+/* TABLET PORTRAIT */
+@media screen and (min-width: 768px) {
```
Comment (Skorcese) [inline style.css:95]: @media screen and (min-width: 768px) and (orientation: portrait) { ... }

File: style.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    }
+
+    /* HERO - LINK */
+    .hero__link {
+        font-size: 1.5em;
+    }
+}
+
+/* TABLET LANDSCAPE */
+@media screen and (min-width: 992px) {
```
Comment (Skorcese) [inline style.css:119]: @media screen and (min-width: 992px) and (orientation: landscape) { ... }

**Review by Skorcese — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kacus) [reply index.html:14]: solved in #feee4f40d19ce893160a45df59c315c715572e10

File: style.css (hunk 3 above)
Comment (kacus) [reply style.css]: Solved in https://github.com/kacus/CC-porfolio/pull/1/commits/28f4bdbb60da2ef0c06a166ff99fd95f88609892

File: style.css (hunk 4 above)
Comment (kacus) [reply style.css]: Solved in https://github.com/kacus/CC-porfolio/pull/1/commits/28f4bdbb60da2ef0c06a166ff99fd95f88609892

File: style.css (hunk 5 above)
Comment (kacus) [reply style.css]: Solved in https://github.com/kacus/CC-porfolio/pull/1/commits/28f4bdbb60da2ef0c06a166ff99fd95f88609892

**Pull request merged**